homepage/pj1.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+    <link rel="stylesheet" href="styles.css" type="text/css">
+ </head>
+<body>
+    <div id="main">
+        <div class="header">
+            <div class="nav">
+                    <ul>
+                        <section class="nav-left">
+                            <li><a href="#home">Home</a></li>
+                            <li><a href="#about">About</a></li>
+                            <li><a href="#practical">Practice Areas</a></li>
+                            <li><a href="#News"></a>News</li>
```

Comment:
Why is the named anchor starts with Capital letter but not others? consistency matters